repairnator/repairnator-scanner/src/main/java/fr/inria/spirals/repairnator/scanner/Launcher.java

```diff
@@ -263,7 +270,11 @@ private void printToFile(List<BuildToBeInspected> listOfBuilds) {
 
     private void printToStdout(List<BuildToBeInspected> listOfBuilds) {
         for (BuildToBeInspected buildToBeInspected : listOfBuilds) {
-            System.out.println(buildToBeInspected.getBuggyBuild().getId());
+            if (this.config.getLauncherMode() == LauncherMode.REPAIR) {
+                System.out.println(buildToBeInspected.getBuggyBuild().getId());
+            } else {
+                System.out.println(buildToBeInspected.getBuggyBuild().getId() + Utils.COMMA + buildToBeInspected.getPatchedBuild().getId());
```

Comment:
We'll have the same error as before here: getId() return an int and comma is a char, so it will do the addition instead of concatenate the string

Reply:
Fixed.